### ISSUE TITLE
Able to submit even when all grid answers are not completed

### ISSUE THREAD

**Issue by cunnaneaq** [episphere/questionnaire#101]
For the grid questions in mod 2 and the biospecimen module, you are able to submit the page without answering all of the questions in the grid.  

Is it possible to have a "Answer the question" "Continue without answer" modal pop up?

**Comment by cusackjm** [comment]
@naiyume is this a quest or markup issue?

**Comment by joshid-ims** [comment]
Is the pop up not showing if none are answered or some are not answered? Also is it showing for Module 3?

**Comment by joshid-ims** [comment]
@cunnaneaq should the pop up show for all questions? 

Currently, if user answers one question, pop up is not shown.

Also, not sure if this is something Daniel can add as a functionality.

**Comment by cusackjm** [comment]
@joshid-ims the pop up shows up if none are answered, but doesn't show up once one is answered

**Comment by joshid-ims** [comment]
Yes, exactly. So should it should up for all unanswered ones in grid? That might be something Daniel will need to add.

**Comment by joshid-ims** [comment]
@cusackjm I think Daniel will need to confirm this.

**Comment by naiyume** [comment]
@cusackjm we have pushed a fix for this. Could you check on it?

**Comment by cusackjm** [comment]
confirmed fix during testing on 5/17/22 in ivaniteb5 CONNECT ID: 9052791267
